sdk/resourcemanager/azure-resourcemanager-authorization/src/main/java/com/azure/resourcemanager/authorization/implementation/ServicePrincipalImpl.java

```diff
@@ -166,15 +189,13 @@ private Mono<ServicePrincipal> submitRolesAsync(final ServicePrincipal servicePr
             create =
                 Flux
                     .fromIterable(rolesToCreate.entrySet())
-                    .flatMap(
-                        roleEntry ->
-                            manager()
-                                .roleAssignments()
-                                .define(this.manager().internalContext().randomUuid())
-                                .forServicePrincipal(servicePrincipal)
-                                .withBuiltInRole(roleEntry.getValue())
-                                .withScope(roleEntry.getKey())
-                                .createAsync())
+                    .flatMap(roleEntry -> manager()
```

Comment:
pure format change?

Reply:
yes (because previously try to move the retry on RBAC here, but find out there are MSI created in other resources that require retry on RBAC)